### DISCUSSION TITLE
Amino Acid Mismatches to Database Reference Sequence

### DISCUSSION THREAD

**Discussion by DarioS** [vdemichev/DiaNN#155] [General]
For a peptide sequence, how many mismatches to a UniProt FASTA file does DIA-NN tolerate? Someone who is experienced in proteomics research recently mentioned to me that the popular convention is to search for exact matches only. That doesn't seem suitable if analysing, for instance, melanoma, which is the most highly (missense) mutated cancer amongst all of the TCGA cancer types. I can't see any discussion of this matter in the user guide and perhaps it would make a nice addition to it.

**Comment by vdemichev** [comment]
Normally yes, exact matches. It's a drawback of DIA that sometimes mismatched sequences will also get identified, e.g. PEPTIDE can be matched to a spectrum originating from PQPTIDE. This will not happen too often though. To significantly reduce the rate of such mismatches, if they are possible due to the nature of the sample, can use filtering based on Ms1.Profile.Corr and Mass.Evidence. If, in contrast, you specifically want mismatches to happen, then there's currently no way to increase their chance.

**Reply by DarioS** [reply]
I was reading [The Human Melanoma Proteome Atlas—Complementing the Melanoma Transcriptome](https://onlinelibrary.wiley.com/doi/10.1002/ctm2.451) today and they make it look straightforward. I hope it can be a future enhancement.

![image](https://user-images.githubusercontent.com/631218/142176757-db94331d-b2c2-45ec-9b37-de83a51783cd.png)

**Reply by vdemichev** [reply]
Yes, if I understood correctly, they included these mutations (a very limited number) in the FASTA database. So it's not like the software was identifying them de-novo. Also, the FDR for these when identified from DIA data might not be dramatically high, but would definitely be higher than for regular peptides.
